src/fragments/lib/auth/common/sms/flows.mdx

```diff
@@ -1,3 +1,10 @@
+<Callout>
```

Comment:
does this render okay? with components in MDX we typically need to surround the markdown content in new lines, see https://github.com/aws-amplify/docs/blob/d1b859e38e03e174a022423957909ac7ab90a51e/src/fragments/lib/auth/common/sms/flows.mdx#setup

Reply:
@josefaidt Yes. It does render correctly. 
![image](https://user-images.githubusercontent.com/88387035/164770555-0a5f8dd1-6541-4324-a6f7-ff42e2ba1da3.png)